.circleci/config.yml

```diff
@@ -1,7 +1,27 @@
 version: 2.1
-orbs:
-  hello: circleci/hello-build@0.0.5
+
+jobs:
+  frontend-test:
+    docker:
+      - image: circleci/node:16.4
+    working_directory: ~/pin-my-hike/client
```

Comment:
Nice that it works in the client, does the root/server.js need to be configured as well?